app/_src/gateway/kong-manager/auth/oidc/configure.md

```diff
@@ -18,7 +18,8 @@ OIDC for Kong Manager.
 The following is an example using Google as the IdP and serving Kong Manager
 from its default URL, `http://127.0.0.1:8002`.
 
-The `admin_gui_auth_config` value must be valid JSON.
+{% if_version lte:3.4.x %}
```

Comment:
@raoxiaoyan Hi! I added some conditional rendering since I think the changes only apply to 3.5.x and greater, right? If that sounds correct, I can approve and merge this :)

Reply:
Correct. Thank you.